requirements/base.in

```diff
@@ -2,46 +2,49 @@
 # like Pillow and psycopg2
 # See http://bitly.com/wheel-building-fails-CPython-35
 # Verified bug on Python 3.5.1
-wheel==0.29.0
+wheel
 
-django==1.9.4
+django
```

Comment:
Not specifying the versions is AFAIK the right way to do this since `pip-tools` requires you to use a special flag to upgrade requirements: `-U, --upgrade Try to upgrade all dependencies to their latest versions`